### DISCUSSION TITLE
Button to switch between html/plain text

### DISCUSSION THREAD

**Discussion by blogdemoi** [the-djmaze/snappymail#1633] [Q&A]
Hi,
Thank you for Snappymail. Is there a way to add a button to quickly switch between html/plain text e-mails?
Thank you.

**Comment by the-djmaze** [comment]
There is in the menu.
Is that not sufficient?

**Reply by blogdemoi** [reply]
> There is in the menu. Is that not sufficient?

Yes, I know, that is what I use now. A button would be nice, at least for my usage... I understand this is not something everybody is requesting though. Anyway, thank you for Snappymail, very useful.
